### DISCUSSION TITLE
Files with LBC extension are not categorized as audios

### DISCUSSION THREAD

**Discussion by Jefresongil** [sepinf-inc/IPED#1797] [General]
Processando uma base de dados contendo áudios oriundos de Interceptações Telefônicas, percebi que o IPED não está catalogando arquivos com extensão .lbc conforme imagem abaixo na categoria Áudios.

Estes arquivos são executados pelo Software Smplayer.exe https://www.smplayer.info/pt_BR/downloads

Atualmente tenho a versão 4.1.2 instalada.

![image](https://github.com/sepinf-inc/IPED/assets/120651461/538dc97e-adb5-4437-9cf1-cc2542f5cd4e)

**Comment by lfcnassif** [comment]
Hi. Not sure but maybe your audios are related to `audio/ilbc` mimetype. Please check into SMPlayer if there is some information about one of your audios confirming that. If yes, we can at least classify those audios based on the `lbc` extension. Or if you can share one sample privately, that would help.

A better classification would take into account the audio format signature (starting bytes) to avoid depending just on the `lbc` extension. Into IPED's Hex viewer, if you can identify some byte sequence/pattern present at the start of all your audios, let me know, that would make classification robust against renamed files.

**Reply by Jefresongil** [reply]
Sorry for the delay. I hadn't seen the message.

Hexadecimal pattern obtained from IPED. 
I tried to attach a file but GitHub didn't accept the extension.

![Padrão Hexadecimal arquivo extensão  lbc](https://github.com/sepinf-inc/IPED/assets/120651461/6febaef7-2d57-4444-bd38-17f2c29fb545)

**Reply by lfcnassif** [reply]
Thanks @Jefresongil. @tc-wleite added the support to LBC audios recently, please try the latest snapshot on your audios:
https://github.com/sepinf-inc/IPED/actions/runs/5814178572

**Comment by wladimirleite** [comment]
I guess the audios are in the iLBC format as @lfcnassif said.
It is not a very common format, but it is supported by MPlayer (used by IPED in the transcription pipeline).
@Jefresongil, I couldn't find any samples here, but if you can send me privately one of these audios, I can add the support to them.

**Comment by wladimirleite** [comment]
I found some LBC audios here.

**Comment by lfcnassif** [comment]
Hi @Jefresongil, you can find a snapshot version with the iLBC audio support here, thanks to @tc-wleite:
https://github.com/sepinf-inc/IPED/actions/runs/5770765083

Please test it and let us know the results.
